spec/truffle/identity_spec.rb

```diff
@@ -8,25 +8,80 @@
 
 require_relative '../ruby/spec_helper'
 
-# Require something that uses code from RubySL
-require 'stringio'
+require 'rubygems'
 
 describe "Identifying features such as" do
 
-    describe "the Rubinius module" do
+  it "RUBY_ENGINE is 'truffleruby'" do
+    RUBY_ENGINE.should == 'truffleruby'
+  end
 
-      it "is not defined" do
-        defined?(Rubinius).should be_nil
-      end
-      
+  it "RUBY_VERSION matches /\A\d+\.\d+\.\d+\Z/" do
+    RUBY_VERSION.should =~ /\A\d+\.\d+\.\d+\Z/
```

Comment:
`\Z` allows a trailing `\n` which we don't want here, so it should be `\z`.

Reply:
Fixed.